Formula/v/virtualfish.rb

```diff
@@ -92,7 +92,10 @@ def caveats
     assert_match "new_virtualenv", shell_output('fish -c "vf ls"')
 
     # Delete the virtualenv
-    system "fish", "-c", "vf rm new_virtualenv"
-    refute_path_exists testpath/".virtualenvs/new_virtualenv/pyvenv.cfg"
+    # cannot delete virtualenv on sequoia, upstream bug report, https://github.com/justinmayer/virtualfish/issues/250
+    if OS.mac? && MacOS.version < :sequoia
+      system "fish", "-c", "vf rm new_virtualenv"
+      refute_path_exists testpath/".virtualenvs/new_virtualenv/pyvenv.cfg"
+    end
```

Comment:
```suggestion
    return if OS.mac? && MacOS.version >= :sequoia

    system "fish", "-c", "vf rm new_virtualenv"
    refute_path_exists testpath/".virtualenvs/new_virtualenv/pyvenv.cfg"
```
Plus adjust the comment placement, probably